wagtailio/packages/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class PackagesConfig(AppConfig):
```

Comment:
Missing ``default_app_config`` in ``__init__.py``

Reply:
We should [not use `default_app_config` anymore](https://docs.djangoproject.com/en/3.0/ref/applications/#configuring-applications) and instead add the app's config dotted path to the `INSTALLED_APPS`, in this case `packages.app.PackagesConfig`.

Reply:
I removed the `PackagesConfig `. It is required if we want to config the 'application', which we don't.